package.json

```diff
@@ -41,7 +41,10 @@
 		"commands": {
 			"postlink": "node node_modules/react-native-geth/scripts/rnpm-postlink"
 		}
-	},
+  },
+  "resolutions": {
+    "**/eslint-utils": "^1.4.1"
```

Comment:
Seems like the latest ESlint uses a patched version of eslint-utils. forcing a resolution like this might work but just upgrading eslint is preferable 
https://github.com/eslint/eslint/blob/master/package.json#L53

Reply:
I've upgraded eslint to the latest version. It fixes the bug as you have mentioned.